php/commands/rewrite.php

```diff
@@ -173,10 +173,22 @@ public function structure( $args, $assoc_args ) {
 	 * : Show rewrite rules from a particular source.
 	 *
 	 * [--fields=<fields>]
-	 * : Limit the output to specific fields. Defaults to match,query,source.
+	 * : Limit the output to specific fields.
+	 * ---
+	 * default: match,query,source
```

Comment:
Do you mind reverting this change for now? I'd like to give it a bit more thought before we commit to a direction.